src/components/SingleReview.jsx

```diff
@@ -0,0 +1,59 @@
+import { useParams } from 'react-router-dom';
+import { getReviewById } from '../utils';
+import { useState, useEffect } from 'react';
+import CategoryBar from './CategoryBar';
+
+const SingleReview = ({
+  isLoading,
+  setIsLoading,
+  categories,
+  setCategories,
+}) => {
+  const [review, setReview] = useState([]);
+
+  const { review_id } = useParams();
```

Comment:
Nice to see you're using useParams here so the user can go directly to the review 👍